.github/workflows/site.yml

```diff
@@ -1,35 +1,98 @@
-name: Website
-
-env:
-  JDK_JAVA_OPTIONS: -XX:+PrintCommandLineFlags -XX:MaxMetaspaceSize=4G -Xmx4G -Xss8M # JDK_JAVA_OPTIONS is _the_ env. variable to use for modern Java
-  JVM_OPTS: -XX:+PrintCommandLineFlags -XX:MaxMetaspaceSize=4G -Xmx4G -Xss8M # for Java 8 only (sadly, it is not modern enough for JDK_JAVA_OPTIONS)
+# This file was autogenerated using `zio-sbt-website` via `sbt generateGithubWorkflow` 
+# task and should be included in the git repository. Please do not edit it manually.
 
-on:
-  push:
-    branches: [master]
+name: Website
+'on':
+  workflow_dispatch: {}
   release:
     types:
-      - published
-
+    - published
+  push:
+    branches:
+    - series/2.x
+  pull_request: {}
 jobs:
-  publish:
-    runs-on: ubuntu-20.04
-    timeout-minutes: 30
-    if: github.event_name != 'pull_request'
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
+    steps:
+    - name: Git Checkout
+      uses: actions/checkout@v3.3.0
+      with:
+        fetch-depth: '0'
+    - name: Setup Scala
+      uses: actions/setup-java@v3.9.0
+      with:
+        distribution: temurin
+        java-version: 17
+        check-latest: true
+    - name: Check if the README file is up to date
+      run: sbt  docs/checkReadme
+    - name: Check if the site workflow is up to date
+      run: sbt  docs/checkGithubWorkflow
+    - name: Check artifacts build process
+      run: sbt  +publishLocal
+    - name: Check website build process
+      run: sbt docs/clean; sbt  docs/buildWebsite
```

Comment:
```
docs/clean
```
Is this really needed? Clean after what?

Reply:
Yes, we need that. The `docs/buildWebsite` needs a clean state.